sport/test/FootballTestData.scala

```diff
@@ -163,23 +163,31 @@ object FootballTestData {
     ),
   )
 
-  private def liveMatch(homeName: String, awayName: String, homeScore: Int, awayScore: Int, date: ZonedDateTime) =
+  def liveMatch(
+      homeName: String,
+      awayName: String,
+      homeScore: Int,
+      awayScore: Int,
+      date: ZonedDateTime,
+      isLive: Boolean = true,
```

Comment:
would it be safer to not have a default for this? or to default to false? Intuitively if we don't get information about whether a match is live I assume it's more likely that it isn't live, but I may be misunderstanding